src/turtle.rs

```diff
@@ -199,6 +199,46 @@ impl Turtle {
         block_on(self.turtle.wait(secs))
     }
 
+    /// Instructs the turtle to draw an arc by the given angle for a given radius in the given step count from the current position of the turtle.
+    ///
+    /// The `radius` parameter is a floation point number that represents the radius of the arc
+    /// that you want to draw. If a negative `radius` is passed its absolute value is
+    /// considered for drawing the arc.
+    ///
+    /// The `extent` parameter is a floating point number that represents how much you want the
+    /// turtle to rotate. If the `extent` parameter is `None` then it is defaulted to 360. If
+    /// a negative `extent` is passed the arc is drawn in the opposite direction.
```

Comment:
```suggestion
    /// The `extent` parameter is an angle that represents how much of the circle to draw. 
    /// If `None`, the entire circle will be drawn. If negative, the arc will be drawn in
    /// the opposite direction.
```

Instead of specifying `360`, I used "the entire circle" since the actual default value should depend on what the current angle unit is set to.